src/ui/views/convadd.coffee

```diff
@@ -82,22 +82,27 @@ module.exports = view (models) ->
         ul ->
             convsettings.selectedEntities.forEach (r) ->
                 cid = r?.id?.chat_id
-                li class: 'selected', ->
-                    drawAvatar(cid, viewstate, entity)
+                email = (r.properties?.email?[0] ? entity[cid]?.email?[0]);
```

Comment:
shouldn't here be also emails in the second one?:

`email = (r.properties?.email?[0] ? entity[cid]?.emails?[0]); `